account_commission/views/report_settlement_templates.xml

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="report_settlement" inherit_id="commission.report_settlement">
-        <xpath expr="//th" position="before">
-            <th>Invoice date</th>
-            <th>Invoice</th>
-            <th>Invoice line</th>
+    <template
+        id="report_settlement_document"
+        inherit_id="commission.report_settlement_document"
+    >
+        <xpath expr="//th" position="after">
```

Comment:
These changes should act only if the settlement type is from invoices.

Reply:
done, also (re)added the settlement invoice but in the information area:
![account_commission_after_with_settlement_invoice](https://github.com/OCA/commission/assets/50988819/2139461d-a645-4f6c-aeb9-b7f520ae8e99)